package.json

```diff
@@ -55,15 +59,20 @@
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^3.3.1",
     "loader.js": "^4.7.0",
+    "mapbox-gl": "^1.13.1",
     "npm-run-all": "^4.1.5",
     "prettier": "^2.2.1",
     "qunit": "^2.14.1",
     "qunit-dom": "^1.6.0"
   },
   "engines": {
-    "node": "10.* || >= 12"
+    "node": "12.18.2"
```

Comment:
Can you explain why you modified this?

Reply:
I modified it to deploy application on heroku. Because ember buildpack which used for heroku deployment was not compatible with other node versions.
I tried different node versions but deployment failed, After R&D I found that people are using this particular version to deploy ember application on heroku.